data/strings/strings.txt

```diff
@@ -34695,166 +34840,1691 @@
+    es-MX = Cargando su archivo…
+    sv = Laddar upp din fil…
+    th = กำลังอัพโหลดไฟล์ของคุณ…
+    tr = Dosyanız yükleniyor…
+    uk = Завантажуємо ваш файл…
+    vi = Đang tải tập tin của bạn…
+    ei = Ανεβάζουμε το αρχείο σας…
+    sk = Nahrávanie váľho súboru…
+    sw = Inapakia faili lako…
+    zh-Hans = 正在上传您的文件......
+    zh-Hant = 正在上傳您的文件......
 
   [direct_link_success]
     en = The link has been created
     ru = Ссылка создана
+    ar = لقد تم إنشاء الرابط
+    cs = Odkaz byl vytvořen
+    da = Linket er blevet oprettet
+    nl = De link is aangemaakt
+    fi = Linkki on luotu
+    fr = Le lien a été créé
+    de = Link erstellt
+    hu = A hivatkozás létre van hozva
+    id = Tautan telah dibuat
+    it = Il link è stato creato
+    ja = リンクが作成されました
+    ko = 링크가 만들어졌습니다
+    nb = Lenken er opprettet
+    pl = Link został utworzony
+    pt = Link criado
+    pt-BR = O link foi criado
+    ro = Linkul a fost creat
+    es = Enlace creado
+    es-MX = El enlace fue creado
+    sv = Länk skapad
+    th = ลิงค์ถูกสร้างแล้ว
+    tr = Bağlantı oluşturuldu
+    uk = Посилання створено
+    vi = Liên kết đã được tạo
+    ei = Ο σύνδεσμος δημιουργήθηκε
+    sk = Odkaz bol vytvorený
+    sw = Kiunganishi kimeundwa
+    zh-Hans = 链接已创建
+    zh-Hant = 鏈接已創建
 
   [edit_guide_title]
-    en = Edit your guide on the web
-    ru = Редактируйте ваш путеводитель в интернете
+    en = Edit your guide on web
+    ru = Редактируйте ваш путеводитель онлайн
+    ar = قم بتحرير دليلك على الويب
+    cs = Upravujte váš průvodce online
+    da = Rediger din guide på web
+    nl = Bewerk uw gids op internet
+    fi = Muokkaa matkaoppaasi verkossa
+    fr = Éditez votre guide sur le web
+    de = Bearbeiten Sie Ihr Reiseführer online
+    hu = Szerkessze az útikalauzát online
+    id = Edit panduan Anda di web
+    it = Modificare la tua guida online
+    ja = ガイドをウェブ上で編集
+    ko = 웹에서 당신의 가이드 수정하기
+    nb = Rediger din guide på nettet
+    pl = Edytuj swój przewodnik online
+    pt = Edite seu guia de viagens on-line
+    pt-BR = Editar seu guia online
+    ro = Redactați ghidul dvs online
+    es = Edite su guía en línea
+    es-MX = Edite su guía en la web
+    sv = Redigera din reseguide online
+    th = แก้ไขไกด์ของคุณบนเว็บ
+    tr = Kılavuzunuzu web üzerinde düzenleyin
+    uk = Редагуйте ваш путівник онлайн
+    vi = Chỉnh sửa hướng dẫn của bạn trên trang web
+    ei = Επεξεργαστείτε τον οδηγό σας στο δίκτυο
+    sk = Upravte vášho sprievodcu na webe
+    sw = Hariri mwongozo wako katika mtandaoni
+    zh-Hans = 请在线编辑您的旅行指南
+    zh-Hant = 請在線編輯旅行指南
 
   [edit_guide_subtitle]
-    en = Get a link to edit your guide via web app: add descriptions for POIs, upload photos and more
-    ru = Получите ссылку для редактирования вашего путеводителя через веб-приложение: добавляйте описания к меткам, загружайте фотографии и многое другое
+    en = Get a link to edit your guide via web app: add descriptions, upload photos and connect bookmarks with lines.
```

Comment:
@therearesomewhocallmetim oxford comma?

Reply:
@mpimenov imo, in this particular case it doesn't affect the meaning